docs_user/modules/proc_migrating-the-rgw-backends.adoc

```diff
@@ -165,11 +160,13 @@ Query 172.17.3.81
 ---
 ----
 
-. If RGW backends are migrated in the {Ceph} nodes, there is no "`internalAPI`" network(this is not true in the case of HCI). Reconfigure the RGW keystone endpoint, pointing to the external network that has been propagated. For more information about propagating the external network, see xref:completing-prerequisites-for-migrating-ceph-rgw_{context}[Completing prerequisites for migrating {Ceph} RGW].
+. If RGW back ends are migrated to the {Ceph} nodes, there is no "`internalAPI`" network. You must reconfigure the RGW keystone endpoint to point to the external network that you propagated:
```

Comment:
Maybe
"If you migrated RGW back ends to the {Ceph} nodes, there is no "`internalAPI`" network."
to make it less passive.
Why was the HCI exception (this is not true in the case of HCI) dropped?
Why is there double quotes && backticks?